dojo/src/tests/test_world.cairo

```diff
@@ -1,83 +0,0 @@
-#[cfg(test)]
-mod tests {
```

Comment:
Deleted this file since this tests seems to be outdated, pointing to models that do not exists